src/highlight/HighlightList.tsx

```diff
@@ -1,20 +1,48 @@
 import React from 'react';
 import classNames from 'classnames';
-import HighlightAnnotation from './HighlightAnnotation';
+import HighlightCanvas from './HighlightCanvas';
+import HighlightSvg from './HighlightSvg';
+import HighlightTarget from './HighlightTarget';
 import useOutsideEvent from '../common/useOutsideEvent';
-import { AnnotationHighlight } from '../@types';
-
+import { AnnotationHighlight, Rect } from '../@types';
+import { checkValue } from '../utils/util';
 import './HighlightList.scss';
 
 export type Props = {
-    annotations: AnnotationHighlight[];
+    activeId?: string | null;
+    annotations: Pick<AnnotationHighlight, 'id' | 'target'>[];
     className?: string;
     onSelect?: (annotationId: string | null) => void;
 };
 
-export function HighlightList({ annotations, className, onSelect }: Props): JSX.Element {
+export function isValidHighlight({ target }: Pick<AnnotationHighlight, 'target'>): boolean {
```

Comment:
Can we reduce the duplication of `Pick<AnnotationHighlight, 'target'>`? Seems like it would be simpler to import `TargetHighlight`? Also, can we name this `filterHighlight` to stay consistent with RegionList?